.pre-commit-config.yaml

```diff
@@ -59,3 +80,5 @@ repos:
     -   id: detect-secrets
         args: ['--baseline', '.secrets.baseline']
         exclude: package.lock.json
+ci:
+    autoupdate_schedule: monthly
```

Comment:
So this updates the version of precommit used every month?

Reply:
It updates the pre-commit hooks I believe

Reply:
got it